include/json/config.h

```diff
@@ -80,8 +80,8 @@ extern JSON_API int msvc_pre1900_c99_snprintf(char* outBuf, size_t size,
 // These Macros are maintained for backwards compatibility of external tools.
 #if (defined(_MSC_VER) && _MSC_VER >= 1900) ||                                 \
     (defined(__GNUC__) && __cplusplus >= 201103L) ||                           \
-    (defined(__clang__) && __clang_major__ == 3 && __clang_minor__ >= 3)
 
+    (defined(__clang__) && (__clang_major__ > 3 || (__clang_major__ == 3 && __clang_minor__ >= 3)))
```

Comment:
Need to remove the blank line above this line.
I think the blank line is going to ruin the continuity of the `#if`, and this won't do what it is supposed to do.

Reply:
Of course. I just realized and was about to modify it.